### PR TITLE
Reverse code-search versions order

### PR DESCRIPTION
**Pull request by wagnerand**
Fix mozilla/addons#8495.

### CODE REVIEW

File: src/olympia/scanners/tasks.py [hunk 1]
```diff
@@ -336,7 +336,7 @@ def run_yara_query_rule(query_rule_pk):
     )
     if not rule.run_on_disabled_addons:
         qs = qs.exclude(addon__status=amo.STATUS_DISABLED)
-    qs = qs.values_list('id', flat=True).order_by('pk')
+    qs = qs.values_list('id', flat=True).order_by('-pk')
```
Comment (willdurand) [inline src/olympia/scanners/tasks.py:339]: Do you think this should be an option when creating the query?

Comment (wagnerand) [reply src/olympia/scanners/tasks.py:339]: No. For a complete result, you need to wait for the task to finish anyway, this just helps me to get a head start.

**Comment by willdurand** [comment]
Why would the `release-master` job run in this PR? 🤔

**Comment by eviljeff** [comment]
> Why would the `release-master` job run in this PR? 🤔

@wagnerand is trying to merge from wagnerand:master so it triggered.

**Review by willdurand — APPROVED** [review]
If that works for you, that works for me.

**Pull request merged**